content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -10,6 +10,12 @@ tocDepth: 3
 
 Prisma metrics give you a detailed insight into how Prisma Client interacts with your database. You can use this insight to help diagnose performance issues with your application.
 
+<Admonition type="info">
+
+If you want an even more detailed insight into your application's performance, at the level of individual queries, see [Tracing](/concepts/components/prisma-client/tracing).
```

Comment:
```suggestion
If you want an even more detailed insight into your Clients performance, at the level of individual queries, see [Tracing](/concepts/components/prisma-client/tracing).
```

It's more your Prisma Client and not your application, there could be many other things going on in there application that we don't trace. 

Not sure what the wording here should be: 

* your Prisma Client's performance
* your Client's performance